lib/IframeHandler.js

```diff
@@ -45,7 +45,9 @@ IframeHandler.prototype.init = function (url) {
   }, this.timeout);
 }
 
-IframeHandler.prototype.messageEventListener = function (e) { 
+IframeHandler.prototype.messageEventListener = function (e) {
+  if (!e.data.hasOwnProperty('accessToken')) return;
```

Comment:
This `IframeHandler` has no idea of the business of the `access_token` or anything related.

I would like to handle this some place else. Do you think you can do that?